### PR TITLE
Add ScalarDB SQL (JDBC) Sample

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by josh-wong — CHANGES_REQUESTED** [review]
The tutorial is looking good! I've made a few suggestions, so PTAL!

File: scalardb-sql-jdbc-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+```shell
+$ docker-compose up -d
+```
+
+Please note that starting the containers may take more than one minute.
+
+## Load schema
+
+You then need to apply the schema with the following command.
+To download the schema tool, `scalardb-sql-cli-<VERSION>-all.jar`, see the [Releases](https://github.com/scalar-labs/scalardb-sql/releases) of ScalarDB SQL and download the version that you want to use.
```
Comment (brfrn169) [inline scalardb-sql-jdbc-sample/README.md:122]:
```suggestion
To download the CLI tool, `scalardb-sql-cli-<VERSION>-all.jar`, see the [Releases](https://github.com/scalar-labs/scalardb-sql/releases) of ScalarDB SQL and download the version that you want to use.
```

**Comment by brfrn169** [comment]
@josh-wong Thank you for the suggestions! I have applied them. Please take a look again when you have time!

**Review by josh-wong — APPROVED** [review]
LGTM👍 Thank you!

**Comment by feeblefakie** [comment]
@reddikih Can you verify if it is OK? (I'll merge it for now, so please create a PR if it needs to be revised.)

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**